dotnet/src/Connectors/Connectors.Memory.Postgres/PostgresVectorStoreDbClient.cs

```diff
@@ -172,21 +174,23 @@ public async Task DeleteAsync<TKey>(string tableName, string keyColumn, TKey key
     }
 
     /// <inheritdoc />
-    public async IAsyncEnumerable<(Dictionary<string, object?> Row, double Distance)> GetNearestMatchesAsync(
-        string tableName, IReadOnlyList<VectorStoreRecordProperty> properties, VectorStoreRecordVectorProperty vectorProperty, Vector vectorValue, int limit,
-        VectorSearchFilter? filter = default, int? skip = default, bool includeVectors = false, [EnumeratorCancellation] CancellationToken cancellationToken = default)
+#pragma warning disable CS0618 // VectorSearchFilter is obsolete
```

Comment:
I wonder if there's another non obsolete API that could be leveraged instead of ignoring the obsolescence, or keeping a tracking issue to tackle this later on.

Reply:
The obsolescence here is specifically for the existing (legacy) filter, which we decided to preserve for now, to avoid breaking existing users - the plan is to remove that API (and rename NewFilter to Filter) before we actually release. So all these suppressions will go away in the next 2-3 months.